.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "dockerComposeFile": [
+    "../docker-compose.yml",
+    "docker-compose.yml"
+  ],
+  "service": "devcontainer",
+  "runServices": [
+    "devcontainer"
+  ],
+  "shutdownAction": "none",
```

Comment:
This is what prevents the devcontainer from shutting down when we close out of VS Code IIUC.

Reply:
Yes, because we had issues when it shut the container down.